pyiron_base/generic/dynamic.py

```diff
@@ -0,0 +1,60 @@
+import os
+import json
+from pyiron_base.state import state
+
+
+def _get_class_path_lst():
+    path_lst = [
+        os.path.abspath(os.path.expanduser(os.path.join(p, "template")))
+        for p in state.settings.resource_paths
+        if os.path.exists(p) and "template" in os.listdir(p)
+    ]
+    class_path_lst = []
+    for path in path_lst:
+        class_path_lst += [
+            cp
+            for cp, files in [
+                [cp, os.listdir(cp)]
+                for cp in [os.path.join(path, c) for c in os.listdir(path)]
+            ]
+            if "script.py" in files and "input.json" in files
```

Comment:
If the script does not accept inputs, then it does not really make sense to execute it multiple time - maybe to sample a statistical distribution, but this can be done once and does not need a pyiron job. Based on this my assumption is that each script requires some inputs and generates some outputs and this can be as one or two variables.